website/source/docs/guides/deployment.html.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: "docs"
+page_title: "Production Deployment"
+sidebar_current: "docs-guides-deployment"
+description: |-
+  Best practice approaches for Consul production deployments.
+---
+
+# Consul Production Deployment Guide
+
+As applications are migrated to dynamically provisioned infrastructure, scaling services and managing the communications between them becomes challenging. Consul’s service discovery capabilities provide the connectivity between dynamic applications. Consul also monitors the health of each node and its applications to ensure that only healthy service instances are discovered. Consul’s distributed runtime configuration store allows updates across global infrastructure.
+
+The goal of this document is to recommend best practice approaches for Consul production deployments. The guide provides recommendations on system requirements, datacenter design, networking, and performance optimizations for Consul cluster based on the latest Consul (1.0.7) release.
+
+This document assumes a basic working knowledge of Consul.
+
+## 1.0 System Requirements
+
+Consul server agents are responsible for maintaining the cluster state, responding to RPC queries (read operations), and for processing all write operations. Given that Consul server agents do most of the heavy lifting, server sizing is critical for the overall performance efficiency and health of the Consul cluster.
+
+The following instance configurations are recommended.
+
+|Size|CPU|Memory|Disk|Typical Cloud Instance Types|
+|----|---|------|----|----------------------------|
+|Small|2 core|8-16 GB RAM|50 GB|**AWS**: m5.large, m5.xlarge|
+|||||**Azure**: Standard\_A4\_v2, Standard\_A8\_v2|
+|||||**GCE**: n1-standard-8, n1-standard-16|
+|Large|4-8 core|32-64+ GB RAM|100 GB|**AWS**: m5.2xlarge, m5.4xlarge|
+|||||**Azure**: Standard\_D4\_v3, Standard\_D5\_v3|
+|||||**GCE**: n1-standard-32, n1-standard-64|
+
+The **small size** instance configuration is appropriate for most initial production deployments, or for development/testing environments. The large size is for production environments where there is a consistently high workload.
+
+~> **NOTE** For high workloads, ensure that the disks support high IOPS to keep up with the high Raft log update rate.
```

Comment:
- "large" workloads?
- Supports a high number of IOPS?
- high could be rapid?

Reply:
Fixed in d7fff15d0